Networking/Networking/Model/ShippingLabel/Packages/WooShippingPackagesResponse.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+import Codegen
+
+/// Represents store options, a list of saved Shipping Label Packages (custom and predefined) for the WooCommerce Shipping extension.
+///
+public struct WooShippingPackagesResponse: Equatable, GeneratedFakeable, GeneratedCopiable {
+
+    /// Store options
+    public let storeOptions: ShippingLabelStoreOptions?
```

Comment:
Should I make this non-optional and if the response does not have it we treat is as a failed response?

Reply:
Yes, I'd make it non-optional; I don't think there's any scenario where we'd expect the response to be missing this information, and it's required for our UI, so I'd make it non-optional.